optaplanner-docs/src/main/asciidoc/MoveAndNeighborhoodSelection/MoveAndNeighborhoodSelection-chapter.adoc

```diff
@@ -106,8 +106,46 @@ The root of this tree is a `MoveSelector` which is injected into the optimizatio
 == Generic MoveSelectors
 
 
+[[genericMoveSelectorsOverview]]
+=== Generic MoveSelectors Overview
+
+[cols="1,2a,2a"]
+|===
+|Name |Description |`toString()` example
+
+|<<changeMoveSelector,Change move>>
+|Change 1 entity's variable
+|`Process-A {Computer-1 -> Computer-2}`
+
+|<<swapMoveSelector,Swap move>>
+|Swap all variables of 2 entities
+|`Process-A {Computer-1} <-> Process-B {Computer-2}`
```

Comment:
The "<->" renders quite strangely for me. Would you mind marking it as a pre-formatted text?